data/gegno/gegno outfits.txt

```diff
@@ -819,20 +819,39 @@ outfit "Savagery Pike"
 	cost 21000
 	thumbnail "outfit/savagery pike"
 	"capture attack" .9
-	"capture defense" .8
+	"capture defense" .9
 	"unplunderable" 1
 	description `This is a ceremonial spear-like weapon presented to Vi warriors when they officially become part of the military. On the forward end are several long, sharp pikes that can easily pierce enemies, and on the rear end is a built-in cannon that can fire up to two explosive rounds. The cannon rounds are rarely used by the Vi unless they deem it absolutely necessary to defeat their foe, or to deal with unanticipated outside factors.`
 
+outfit "Irate Carronade"
+	category "Hand to Hand"
+	cost 39000
+	thumbnail "outfit/irate carronade"
+	"mass" 3
+	"outfit space" -3
+	"capture defense" 3.7
+	"unplunderable" 1
+	description `A minaturized version of a ship-to-ship cannon, haphazardly shoved into a small makeshift barricade, isn't exactly the safest practice of installing boarding defenses. Despite this, the hard-armored hulls of larger Gegno warships prove resilient enough to field Irate Carronades. Anyone, or anything, that unknowingly approaches a corridor where one is installed will likely become a new paint layer on the nearby walls.`
+	
+outfit "Plasmasickle"
+	category "Hand to Hand"
+	cost 17000
+	thumbnail "outfit/plasmasickle"
+	"capture attack" .7
+	"capture defense" .6
+	"unplunderable" 1
+	description `Plasmasickles are specialized assault weapons that are very reliant on close-quarters combat. Using a small portion of the Scin's recent plasma technology, the blade is able to be heated to higher temperatures in order to slice through tougher targets. Only select groups of Scin crew are equipped with these, but are trained well enough in assassination to use them effectively.`
```

Comment:
```suggestion
	description `Plasmasickles are specialized assault weapons that are very reliant on close-quarters combat. Using a small portion of the Scin's recent plasma technology, the blade can be heated to higher temperatures in order to slice through tougher targets. Only select groups of Scin crew are equipped with these, but are trained well enough in assassination to use them effectively.`
```
I think 'is able to' and 'in order to' was just too much so close to each other. Just a feeling though, so feel free to ignore this.